docs/setup.md

```diff
@@ -212,19 +212,20 @@ To build Linux, Mac, and Windows wheels using GitHub Actions, create a `.github/
             os: [ubuntu-20.04, windows-2019, macos-10.15]
 
         steps:
-          - uses: actions/checkout@v2
+          - uses: actions/checkout@v3
 
           # Used to host cibuildwheel
-          - uses: actions/setup-python@v2
+          - uses: actions/setup-python@v4
```

Comment:
@mayeut did they forget to update the docs? https://github.com/actions/setup-python#using-python-without-setup-python

Reply:
Oh, never mind. It is not about parameters. Well, I don't know why they would break defaults to favor `pyenv` if it is not used, so let's be explicit.

Reply:
Better stick to v3 until the dust settles down.

Reply:
I'd do that, v3 is fine for now.

Reply:
```suggestion
          - uses: actions/setup-python@v3
```

Reply:
I decided that it is better to remove `setup-python` altogether #1145.

Reply:
Replaced everything with `setup-python@v3`.